src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -255,7 +255,7 @@ public boolean hasGiverRespondedForSession(String userEmail, String feedbackSess
         return viewableResponses;
     }
 
-    public boolean shouldNameBeVisibleToUser(
+    public boolean isNameVisibleToUser(
```

Comment:
I changed this again because I feel that 'is' actually has the same effect as 'should' and would be more consistent with the other method names